Vungle/VungleRouter.m

```diff
@@ -653,14 +657,17 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
 
     if ([self.delegatesDict objectForKey:placementID]) {
         if (isAdPlayable) {
+            MPLogInfo(@"Ad playability update returned ad is playable for Placement ID: %@", placementID);
```

Comment:
You might want to add "Vungle: " here as well?

Reply:
@ashinagawa Got it, thanks a lot for the good suggestion. :)

Reply:
@ashinagawa  Updated, thanks ^_^